src/sass/components/Header.scss

```diff
@@ -0,0 +1,12 @@
+.Header {
```

Comment:
When refreshing the page the blue `Login` button initially is slightly wider than it's automatic width then shrinks. It's only on page load and lasts for <100ms but happens every time.